api/src/pcapi/core/offerers/api.py

```diff
@@ -2737,6 +2740,40 @@ def get_offer_confidence_level(
     return offerer_confidence_level or venue_confidence_level
 
 
+def send_reminder_email_to_individual_offerers() -> None:
+    offerers = (
+        offerers_models.Offerer.query.join(
+            offerers_models.IndividualOffererSubscription,
+            offerers_models.IndividualOffererSubscription.offererId == offerers_models.Offerer.id,
+        )
+        .options(
+            sa.orm.joinedload(offerers_models.Offerer.UserOfferers).joinedload(offerers_models.UserOfferer.user),
+        )
+        .filter(
+            offerers_models.IndividualOffererSubscription.isEmailSent.is_(True),
+            offerers_models.IndividualOffererSubscription.isReminderEmailSent.is_(False),
+            offerers_models.IndividualOffererSubscription.dateEmailSent < date.today() - timedelta(days=30),
+            sa.or_(
+                offerers_models.IndividualOffererSubscription.isCriminalRecordReceived.is_(False),
+                offerers_models.IndividualOffererSubscription.isCertificateReceived.is_(False),
```

Comment:
La condition pour considérer les « documents reçus » (affichage et filtrage) est `offerer.individualSubscription.isCriminalRecordReceived and offerer.individualSubscription.isExperienceReceived`. Je pense donc qu'il ne faudrait pas envoyer le message de relance si seul `isCertificateReceived` est faux.

Reply:
Vu avec Rose : en effet on considère bien que les documents sont reçus si on a casier judiciaire et expérience.